packages/neo4j-driver/test/temporal-types.test.js

```diff
@@ -38,7 +38,7 @@ const MAX_TEMPORAL_ARRAY_LENGTH = 1000
  */
 const MAX_DURATION_COMPONENT = 3000000000000
 const MAX_NANO_OF_SECOND = 999999999
-const MAX_YEAR = 999999999
+const MAX_YEAR = 99_999
```

Comment:
Feels like `packages/bolt-connection/test/bolt/bolt-protocol-v*.test.js` could also make user of this variable instead of hard coding it into the tests.

Reply:
They are different packages, so they could not share test code 😢.